pom.xml

```diff
@@ -13,10 +13,7 @@
     <url>https://wiki.jenkins.io/display/JENKINS/S3+Plugin</url>
 
     <properties>
-        <jenkins.version>1.625.3</jenkins.version>
```

Comment:
In general I prefer to keep the `jenkins.version` on each plugin pom, so that it's easier to know the target of the plugin

Reply:
TBH I have the same opinion but I wasn't sure of which the general approach is.